tools/pylint-extensions/azure-pylint-guidelines-checker/README.md

```diff
@@ -53,12 +53,12 @@ You can identify a custom checker by the link to the guidelines included in its
 | C4757 | do-not-import-legacy-six | Do not import the six package in your library. | # pylint:disable=do-not-import-legacy-six | No Link. | [Example](code_examples.md#do-not-import-legacy-six) |
 | C4758 | docstring-keyword-should-match-keyword-only | Docstring keyword arguments and keyword-only method arguments should match. | # pylint:disable=docstring-keyword-should-match-keyword-only | [Docstrings](https://azure.github.io/azure-sdk/python_documentation.html#docstrings) | [Example](code_examples.md#docstring-keyword-should-match-keyword-only) |
 | C4759 | docstring-type-do-not-use-class | Docstring type is formatted incorrectly. Do not use `:class` in docstring type. | # pylint:disable=docstring-type-do-not-use-class | [Sphinx docstrings](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html) | [Example](code_examples.md#docstring-type-do-not-use-class) |
-| C4760 | no-typing-import-in-type-check | Do not import from typing inside of TYPE_CHECKING. You can import from typing outside of TYPE_CHECKING. | # pylint:disable=no-typing-import-in-type-check | No Link. | [Example](code_examples.md#no-typing-import-in-type-check) |
+| C4760 | no-typing-import-in-type-check | Do not import from typing inside of `if TYPE_CHECKING` block. You can import modules from typing outside of TYPE_CHECKING. | # pylint:disable=no-typing-import-in-type-check | No Link. | [Example](code_examples.md#no-typing-import-in-type-check) |
 | C4761 | do-not-use-legacy-typing | Do not use legacy typing using comments. Python 2 is no longer supported, use Python 3.9+ type hints instead. | # pylint:disable=do-not-use-legacy-typing | No Link. | [Example](code_examples.md#do-not-use-legacy-typing) |
 | C4762 | do-not-log-raised-errors | Do not log errors at error or warning level when error is raised in an exception block | # pylint:disable=do-not-log-raised-errors | No Link. | [Example](code_examples.md#do-not-log-raised-errors) |
```

Comment:
Do we need to update the description for `do-not-log-raised-errors` to exclude the part about error/warning level?